tests/wpunit/ConfigTest.php

```diff
@@ -23,6 +23,24 @@ public function should_set_hook_prefix() {
 		$this->assertEquals( 'bork', Config::get_hook_prefix() );
 	}
 
+	public function paths_provider() {
+		yield 'plugin' => [ WP_PLUGIN_DIR . '/assets/', '/var/www/html/wp-content/plugins/assets/' ];
+		yield 'theme' => [ get_theme_file_path() . '/assets/', get_theme_file_path() . '/assets/' ];
+		yield 'mu-plugin' => [ WPMU_PLUGIN_DIR . '/assets/', '/var/www/html/wp-content/mu-plugins/assets/' ];
+		yield 'content' => [ WP_CONTENT_DIR . '/assets/', '/var/www/html/wp-content/assets/' ];
+		yield 'root' => [ ABSPATH . 'assets/', '/var/www/html/assets/' ];
+		yield 'relative' => [ 'src/resources/', 'src/resources/' ];
```

Comment:
The `::set_path()` is [meant to set the path](https://github.com/stellarwp/assets#configuration) to the root of the plugin (or theme) and `::set_relative_asset_path()` is for setting the path to the assets. Ideally, this test would be reworked like so:

```suggestion
		yield 'plugin' => [ WP_PLUGIN_DIR . '/my-plugin/', '/var/www/html/wp-content/plugins/my-plugin/' ];
		yield 'theme' => [ get_theme_file_path(), get_theme_file_path() ];
		yield 'mu-plugin' => [ WPMU_PLUGIN_DIR . '/my-mu-plugin/', '/var/www/html/wp-content/mu-plugins/my-mu-plugin/' ];
		yield 'content' => [ WP_CONTENT_DIR . '/stuff/', '/var/www/html/wp-content/stuff/' ];
		yield 'root' => [ ABSPATH . 'stuff/', '/var/www/html/stuff/' ];
		yield 'relative' => [ 'my-plugin/', '/var/www/html/wp-content/plugins/my-plugin/' ];
```